adal/adal-rules.pro

```diff
@@ -0,0 +1,60 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in C:\Program Files\Android\android-sdk/tools/proguard/proguard-android.txt
+# You can edit the include path and order by changing the proguardFiles
+# directive in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# Add any project specific keep options here:
+
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+
+##---------------Begin: proguard configuration for Common  --------
+-keep, includedescriptorclasses class com.microsoft.identity.common.exception.** { *; }
+-keep, includedescriptorclasses class com.microsoft.identity.common.internal.broker,* { *; }
+-keep, includedescriptorclasses class com.microsoft.identity.common.internal.cache.* { *; }
+-keep, includedescriptorclasses class com.microsoft.identity.logging.internal.dto.* { *; }
+-keep, includedescriptorclasses class com.microsoft.identity.innternal.logging.* { *; }
+-keep, includedescriptorclasses class com.microsoft.identity.common.internal.providers.* { *; }
+-keep, includedescriptorclasses class com.microsort.identity.common.internal.telemtery.* { *; }
+-keep, includedescriptorclasses class com.microsoft.identity.common.internal.ui,* { *; }
```

Comment:
com.microsoft.device.display is missing. (used by our Webview Activity to support Duo device)

Reply:
Should be done?